ExtinctionMonitorFNAL/Geometry/inc/ExtMonFNALModule.hh

```diff
@@ -31,8 +33,8 @@ namespace mu2e {
     unsigned int nxChips() const;
     unsigned int nyChips() const;
 
-    ExtMonFNALModule(const ExtMonFNALPixelChip& chip, const std::vector<double>& hs)
-      : chip_(chip), sensorHalfSize_(hs)
+    ExtMonFNALModule(const ExtMonFNALPixelChip& chip, const std::vector<double>& hs, const double& gapX, const double& offsetY)
+      : chip_(chip), sensorHalfSize_(hs), chipGapX_(gapX), chipOffsetY_(offsetY)
     {}
     // Required by genreflex persistency
     ExtMonFNALModule() {}
```

Comment:
Please initialize the doubles:

ExtMonFNALModule() : chipGapX_{}, chipOffsetY_{} {}